pype/scripts/transcode.py

```diff
@@ -0,0 +1,176 @@
+"""
+transcode.py -input "input.%04d.exr [1-77]" -name name
+-preset prores422HQ review -output path/to/output/folder
+
+transcode.py -input input.mov -name name
+-preset prores422HQ review -output path/to/output/folder
+
+transcode.py -input input.txt -name name
+-preset concat -output path/to/output/folder
+
+Concatenation *.txt files follows syntax from the FFMPEG demuxer guide;
+https://trac.ffmpeg.org/wiki/Concatenate
+"""
+import os
+import sys
+import argparse
+import tempfile
+
+import clique
+from pype import lib
+
+
+preset_templates = {
+    "prores422HQ": {
+        "args": [
+            "-c:v", "prores_ks",
+            "-profile:v", "3",
+            "-pix_fmt", "yuv422p10le",
+            "-qscale:v", "5",
+            "-codec:a", "pcm_s16le"
+        ],
+        "extension": ".mov"
+    },
+    "proresProxy": {
+        "args": [
+            "-c:v", "prores_ks",
+            "-profile:v", "0",
+            "-pix_fmt", "yuv422p10le",
+            "-qscale:v", "5",
+            "-codec:a", "pcm_s16le",
+            "-vf", "scale=trunc(oh*a/2)*2:trunc(ih/2)"
+        ],
+        "extension": ".mov"
+    },
+    "h264": {
+        "args": ["-crf", "18", "-pix_fmt", "yuv420p"],
+        "extension": ".mov"
+    },
+    "concat": {
+        "args": ["-c", "copy"],
+        "extension": ".mov"
+    }
+}
+
+
+def main(input_path="",
+         preset="",
+         name="",
+         output_path="",
```

Comment:
The output_path is used as an argument, but it is immediately after overridden by other path. And truly it is written to the staging directory.

Reply:
The output path is root directory for all the shots, so before being used in the ffmpeg command, it'll need to be modified with the file name and extension.
Do you want it different somehow? Maybe call it root_path or something?

Reply:
I see, I was thinking it should be used as target for files to be exported to. So if your context in standalonepublisher is at folder `shots` in the project, then files will be exported there. But it was misunderstood.

Reply:
Currently its not referencing any published paths. All transcoded files get dumped to the same folder as the editorial file.